orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/TitusInterestingHealthProviderNamesProcessor.java

```diff
@@ -0,0 +1,44 @@
+package com.netflix.spinnaker.orca.clouddriver.tasks.servergroup;
+
+import com.netflix.spinnaker.orca.front50.model.Application;
+import com.netflix.spinnaker.orca.pipeline.model.Stage;
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class TitusInterestingHealthProviderNamesProcessor implements InterestingHealthProviderNamesProcessor {
+  private static final String TITUS = "Titus";
+  private static final String INTERESTING_HEALTH_PROVIDER_NAMES = "interestingHealthProviderNames";
+
+  @Override
+  public boolean supports(String cloudProvider, Application application, Stage stage) {
+    if (cloudProvider.equalsIgnoreCase(TITUS)) {
+      if (application.platformHealthOnly == Boolean.TRUE && application.platformHealthOnlyShowOverride == Boolean.TRUE) {
+        return true;
+      }
+
+      // checks titus job label
+      Map<String, Object> labels = (Map<String, Object>)stage.getContext().get("labels");
```

Comment:
By convention I think there's a space between the cast and variable ... `(Map<String, Object>) stage`